tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -5,15 +5,42 @@
 import datetime
 from openvino.runtime import Core, Model, PartialShape, Dimension, Layout, Type, serialize
 from openvino.preprocess import PrePostProcessor
+from openvino.tools.benchmark.parameters import parse_args
 
-from .constants import DEVICE_DURATION_IN_SECS, UNKNOWN_DEVICE_TYPE, \
+from .constants import BLOB_EXTENSION, DEVICE_DURATION_IN_SECS, UNKNOWN_DEVICE_TYPE, \
     CPU_DEVICE_NAME, GPU_DEVICE_NAME
 from .logging import logger
 
+import os
 import json
 import re
 import numpy as np
 
+
+def parse_and_check_command_line():
+    args = parse_args()
+    parse_status = True
```

Comment:
Do we need `parse_status` if it is always `True` and exception is raised otherwise?

Reply:
Thank you for pointing that out, it was an artifact left by accident from my direct logic rewrite from C++ version. Removed!